R/bbr.R

```diff
@@ -338,6 +351,18 @@ bbi_help <- function(.cmd_args=NULL) {
 #'
 #' @importFrom yaml read_yaml write_yaml
 #' @importFrom fs dir_exists
+#'
+#' @examples
+#' \dontrun{
+#'
+#' # NONMEM executable found at `/opt/NONMEM/nm75/run`:
```

Comment:
Same comment about it not being confusing what executable this is referring to.  Perhaps "# Top-level of NONMEM 7.5 installation is at `/opt/NONMEM/nm75`:"?

Reply:
addressed